torch/_dynamo/debug_utils.py

```diff
@@ -536,7 +536,19 @@ def same_two_models(gm, opt_gm, example_inputs, only_fwd=False):
         log.warning("Could not generate fp64 outputs")
         fp64_ref = None
 
-    res = run_fwd_maybe_bwd(opt_gm, example_inputs, only_fwd)
+    try:
+        res = run_fwd_maybe_bwd(opt_gm, example_inputs, only_fwd)
+    except Exception as e:
+        # This means that the the minified graph is bad/exposes a different problem.
+        # As we are checking accuracy here, lets log the exception and return True.
+        log.warning(
+            (
+                "While minifying the program in accuracy minification mode,"
+                "ran into a runtime exception which is likely an unrelated issue."
+                f" Logging the exception and moving on. Exception: {e}"
```

Comment:
Can you change this to print an abbreviated version of the exception (maybe the name and a couple frames)? This will spam the console and ends up hiding important information

Reply:
Will do.